addons/events/fnc_registerChatCommand.sqf

```diff
@@ -8,6 +8,7 @@ Parameters:
     _command      - Chat command <STRING>
     _code         - Code to execute after command was entered. <CODE>
     _availableFor - "all", "admin" or "adminLogged" (optional, default: "admin") <STRING>
+    _thisArgs     - Variables available to the function <ANY>
```

Comment:
"Variables" ought to be "Argument(s)" and "the function" is not clear - does it refer to "_code"?

(Put yourself in the shoes of a user reading the documentation for this function for the first time)

Also, add an example.